docs/alternator/compatibility.md

```diff
@@ -16,7 +16,21 @@ provision that - Scylla requires you to provision your cluster. You need
 to reason about the number and size of your nodes - not the throughput.
 
 When creating a table, the BillingMode and ProvisionedThroughput options
-are ignored by Scylla.
+are ignored by Scylla. Tables default to a Billing mode of `PAY_PER_REQUEST`
+and `DescribeTable` API calls will return a value of 0 for the RCUs, WCUs
+and NumberOfDecreasesToday within the response.
+
+## Scan ordering
+
+In DynamoDB, the Hash key (or partition key) determines where the item will
+be stored within DynamoDB's internal storage. Another notable difference between
+DynamoDB and Scylla comes down to the underlying hashing algorithm.
```

Comment:
```suggestion
DynamoDB and Scylla come down to the underlying hashing algorithm.
```

Reply:
@tzach, "comes" was actually correct - "another notable difference ... comes".

Reply:
I got this input from Grammerly, but I trust you more.